am/values.yaml

```diff
@@ -173,6 +173,15 @@ api:
   # https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/#priorityclass
   priorityClassName: ""
 
+  env: []
```

Comment:
Did you see that is an AM file, should we include it in the PR? 🤔

Reply:
Yep, sorry for that. I've missed this from my previous AM PR :(